Governance/tech/conventions.md

```diff
@@ -1,147 +1,295 @@
+````{dropdown} **standard_name** 
+**CF - highly  recommended**<br>
+The [`standard_name`](https://cfconventions.org/Data/cf-conventions/cf-conventions-1.9/cf-conventions.html#standard-name) attribute is used to identify the variable content. A list of the approved standard_names is available [here](http://cfconventions.org/Data/cf-standard-names/current/build/cf-standard-name-table.html).<br>
+It was introduced to provide an objective definition of a variable. The `long_name` attribute was, and still is, useful to provide a description of the variable, but as it is defined ad hoc by the dataset creator it is not possible to use it programmatically to individuate a specific variable, across different files. For example, `standard_name` can be used by online data catalogues to query all datasets that provide a particular variable.<br>
+
+**Requirements**<br>
+* `standard_name` contains no whitespace, underscores are used instead, and is case sensitive.<br>
+* The variable needs to have units which are physically equivalent, if not the same, as the ones listed to be able to use a `standard_name`. 
+* If a variable has a standard_name of region or area_type, it must have value(s) from the permitted lists: [region list](https://cfconventions.org/Data/standardized-region-list/standardized-region-list.html) and [area_type list](http://cfconventions.org/Data/area-type-table/current/build/area-type-table.html).
+
+**Recommendations**<br>
+* The variable definition can be further clarified using [modifiers](https://cfconventions.org/Data/cf-conventions/cf-conventions-1.9/cf-conventions.html#standard-name-modifiers), modifications due to common statistical operations are expressed via the cell_methods attribute.<br>
+
+```{warning}
+Even so, not all variables can be mapped to a standard_name, while this is a required attribute for the CF conventions, it should be left out if not existing. Empty strings are not valid.
```

Comment:
NEVER MAKE UP YOUR OWN STANDARD NAME!!! If it's not defined for your variable you *must not* use this metadata attribute.

Reply:
I thought I felt strongly about this :-)
I swap with your suggestion, but left the Empty string ... because often user interpret `don't add a standard_name` as leave it empty